topics/icrc_25_signer_interaction_standard.md

```diff
@@ -338,6 +338,12 @@ The purpose of the `icrc25_managed_identities` message is for the relying party
 `identities`: A list of identities the user has selected to share with the relying party.
 - `publicKey` (`blob`): The DER-encoded public key associated with the identity, derived in accordance with one of [the signature algorithms supported by the IC](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures). The public key can be used to [derive a self-authenticating principal](https://internetcomputer.org/docs/current/references/ic-interface-spec/#principal).
 - `signature` (`blob`): The signature produced by signing the concatenation of the domain separator `\x13ic-signer-challenge` (UTF-8 encoded) and the challenge with the private key associated with the identity.
+- `delegations` (optional): A list of delegations (as defined by the [IC interface specification, authentication section](https://internetcomputer.org/docs/current/references/ic-interface-spec/#authentication)):
```

Comment:
Maybe rename to `delegation_chain` or just `delegation`, as `delegations` indicates that there could be multiple delegations, and iiuc this is intended for a single delegation from `publicKey` to some other key, possibly via a "multi-step" delegation, right?

Reply:
> this is intended for a single delegation from publicKey to some other key, possibly via a "multi-step" delegation, right?

It is intended to have the exact same semantics as delegations have in the context of a canister call. Hence, why I'm referring to the interface spec here. There is no reason really to limit the delegations array to one entry only. In fact, this might only break with expectations of what developers have when interacting with the IC directly.
So, in short: `delegations` is IMHO appropriate.

Reply:
The interface spec refers to the field as `sender_delegation` (singular) so we will use the singular in the property name as well.